### PR TITLE
Frames update

### PR DESCRIPTION
**Pull request by fabriguespe**
Updating [Frames](https://xmtp.org/docs/build/frames)

### CODE REVIEW

File: docs/build/notifications/react-native.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
```
Comment (daria-github) [inline docs/build/notifications/react-native.md:1]: is this change intended to be part of this same PR?

**Review by rochelle228 — APPROVED** [review]
